ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -347,6 +347,55 @@ def test_cumsum(
     )
 
 
+@handle_test(
+    fn_tree="functional.ivy.cummax",
+    dtype_x_axis_castable=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
Please generate all valid dtypes instead of floats only

Reply:
I have changed dtypes to valid and tested. It is successfully passing checks without any errors.